config/initializers/geocoder.rb

```diff
@@ -1,13 +1,13 @@
 Geocoder.configure(
   # Geocoding options
   timeout: 3,                 # geocoding service timeout (secs)
-  lookup: :google,            # name of geocoding service (symbol)
+  lookup: :google_premier,            # name of geocoding service (symbol)
   # ip_lookup: :freegeoip,      # name of IP address geocoding service (symbol)
   language: :en,              # ISO-639 language code
-  # use_https: false,           # use HTTPS for lookup requests? (if supported)
+  use_https: true,           # use HTTPS for lookup requests? (if supported)
   # http_proxy: nil,            # HTTP proxy server (user:pass@host:port)
   # https_proxy: nil,           # HTTPS proxy server (user:pass@host:port)
-  # api_key: nil,               # API key for geocoding service
+  api_key: ['Rb0Wo5J_CAh4GZYWVzkLWlkIZow=', 'gme-charterforcompassion', 'channel'],               # API key for geocoding service
```

Comment:
should we have this here publicly available?

Reply:
maybe, although the key cannot be used by any other domain